rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -200,6 +200,7 @@ u32 RSXThread::OutOfArgsCount(const uint x, const u32 cmd, const u32 count, cons
 #define case_32(offset, step) \
     case_16(offset, step) \
     case_16(offset + 16*step, step)
+// TODO:: Syphurith: I'm not sure about whether this is correct or missing a break;.
```

Comment:
@Nekotekina So it doesn't need the `break;` for those cases.
Got it and i would remove the comment.